docs/source/getting_started/compatibility.rst

```diff
@@ -61,6 +61,23 @@ should also be synchronized with the server version.
      - 0.2.*
 
 
+Updating Python environment
+---------------------------
+
+Passing from an Ansys version to another (e.g. passing from Ansys 2022 R2 to Ansys 2023 R1), it is needed 
```

Comment:
Updating ?

Reply:
@anslpa The Google Developer Style Guidelines say not to use Latin phrases, such as e.g. or i.e.). I'm not exactly sure what you want to say. When moving from one Ansys release to another, you must update the ``ansys-dpf-core`` package and its dependencies. To get the latest version of the ``ansys-dpf-core`` package, use this code:

Reply:
Thanks !